instrumentation/splunk-otel-systemd-auto-instrumentation/README.md

```diff
@@ -0,0 +1,132 @@
+- [Install and configure](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html) the Splunk
+  OpenTelemetry Collector.
+- Debian or RPM based Linux distribution (amd64/x86_64 or arm64/aarch64).
+- Supported applications running as `systemd` services.
+
+## Installation
+
+The `splunk-otel-systemd-auto-instrumentation` deb/rpm package provides the following files to enable and configure Auto
+Instrumentation agent(s) for `systemd` services:
+- [`/usr/lib/systemd/system.conf.d/00-splunk-otel-javaagent.conf`](#systemd-environment-variables): `systemd` drop-in
+  file to enable and configure environment variables for the Java Auto Instrumentation agent.
+- `/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar`: The [Splunk OpenTelemetry Auto Instrumentation Java
+  Agent](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/splunk-java-otel-distribution.html).
+
+Install this package from [Debian/RPM Package Repositories](
+../../docs/getting-started/linux-manual.md#auto-instrumentation-debianrpm-package-repositories), or manually download
+and install the individual [Debian/RPM Packages](
+../../docs/getting-started/linux-manual.md#auto-instrumentation-debianrpm-packages).
+
+After installation, restart the applicable services or reboot the system to enable the Auto Instrumentation agent(s)
+with the default configuration. Optionally, see [Configuration](#configuration) for details about configuring the
+installed agent(s).
+
+## Configuration
+
+> Before making any changes, it is recommended to check the configuration of the system or individual services for
+> potential conflicts.
+
+- **Java**: See the [Advanced Configuration Guide](
+  https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.html)
+  for details about supported options and defaults for the Java agent. These options can be configured via
+  environment variables or their corresponding system properties after installation.
+
+  > **Configuration Priority**:
+  > 
+  > The Java agent can consume configuration options from one or more of the following sources (ordered from highest to
+  > lowest priority):
+  > 1. Java system properties (`-D` flags) passed directly to the agent. For example,
+  >      ```shell
+  >      JAVA_TOOL_OPTIONS="-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar -Dotel.service.name=my-service"
+  >      ```
+  > 2. Environment variables
+  > 3. Property files
+
+### Systemd Environment Variables
+
+The default [`/usr/lib/systemd/system.conf.d/00-splunk-otel-javaagent.conf`](
+./packaging/00-splunk-otel-javaagent.conf) drop-in file defines the following environment variables to
+enable the installed agent(s) to auto instrument supported Java applications running as `systemd` services:
+
+- `JAVA_TOOL_OPTIONS=-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar`
```

Comment:
Let's also add OTEL_EXPORTER_OTLP_ENDPOINT as a default option in the drop-in file too. Quick question: If there are specific options set from the installers (eg. deployment.environment) or profiling, would the relevant env variable be set here?